web/modules/custom/ai_screening_project/src/Helper/Helper.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace Drupal\ai_screening_project\Helper;
+
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\Core\Logger\LoggerChannel;
+use Psr\Log\LoggerAwareInterface;
+use Psr\Log\LoggerAwareTrait;
+use Psr\Log\LoggerTrait;
+
+/**
+ * A helper class for the module.
+ */
+class Helper implements LoggerAwareInterface {
+  use LoggerAwareTrait;
+  use LoggerTrait;
```

Comment:
Set up logging like this in all your classes.

Reply:
Couldn't get it to work in Hook files for some reason. Probably just a typo of sorts. Won't fix since we are moving away from hux and into events.